scripts/packages.R

```diff
@@ -47,6 +45,29 @@ pacman::p_load(
   # gert  ##to track git moves
 )
 
+pkgs_gh <- c(
+  "newgraphenvironment/fpr",
+  "poissonconsulting/fwapgr",
+  'poissonconsulting/poisspatial',
+  "crsh/citr",
```

Comment:
we don't use citr anymore.  I think it is worth it to try to minimize our dependencies in our reporting so it is great to get familiar with what we actually are using piece by piece and add pkgs as we actually need them

ya - this is built from an old template so you wouldn't know but it will be good to get familiar with all those silly details for things like this all over the place. there are so many